src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1351,6 +1351,24 @@ public function findTaggedServiceIds(string $name, bool $throwOnAbstract = false
         return $tags;
     }
 
+    /**
+     * Find definitions of value-objects by tag.
+     *
+     * @return array<string, array> An array of tags with the tagged classes as key, holding a list of attribute arrays
```

Comment:
note that "class" is not totally accurate here but in practice it should be :)
```suggestion
     * @return array<string, array> An array of tags with the tagged class as key, holding a list of attribute arrays
```